### DISCUSSION TITLE
Keyboard shortcuts don't work

### DISCUSSION THREAD

**Discussion by PicchiarelloMatto** [darkreader/darkreader#10183] [General]
In the Firefox version on Windows, any keyboard shortcut to disable / enable Dark Reader does not work.
With Chrome I used it to associate it with a macro to a mouse button, but now that I've switched to Firefox it doesn't work.

**Comment by Gusted** [comment]
See #10176

**Reply by PicchiarelloMatto** [reply]
Thanks, I had only searched for "shortcuts" and had not found this thread.
